apps/src/feedback.js

```diff
@@ -365,7 +365,8 @@ FeedbackUtils.prototype.displayFeedback = function (options, requiredBlocks,
     icon: icon,
     defaultBtnSelector: defaultBtnSelector,
     onHidden: onHidden,
-    id: 'feedback-dialog'
+    id: 'feedback-dialog',
+    showXButton: !(options.app === 'gamelab' || options.app === 'applab'),
```

Comment:
In general, it's better to have this sort of thing as app-config that gets passed in, instead of shared code doing app checks. How difficult would it be to do something like that here?